### PR TITLE
improved documentation and fix on include_dir.

### PR DESCRIPTION
**Pull request by NickNaso**
Fixes https://github.com/nodejs/node-api-headers/issues/3
Added more documentation about **node-api-headers**
Fixed the `include_dir` property.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 const path = require('path');
 const symbols = require('./symbols')
 
-const include_dir = path.relative('.', __dirname, 'include');
+const include_dir = path.relative('.', `${__dirname}/include`);
```
Comment (KevinEady) [inline index.js:6]: Can you elaborate on this change a bit? We don't think this is necessary.

Comment (NickNaso) [reply index.js:6]: `path.relative(from, to)` takes only two prameters. See https://nodejs.org/dist/latest-v16.x/docs/api/path.html#path_path_relative_from_to

Comment (KevinEady) [reply index.js:6]: 🤦  i was thinking `path.resolve` Thanks!

**Review by KevinEady — APPROVED** [review]
LGTM!

**Comment by KevinEady** [comment]
Would using `path.resolve` be better actually? 🤷

File: README.md [hunk 2]
```diff
@@ -3,15 +3,69 @@
...
+This property is a string that represents the include path for the Node-API
+headers.
+
+### `symbols`
+
+This property is an object that represents the symbols exported by Node-API
+grouped by version and api types.
+
+```js
+    V1: {
```
Comment (KevinEady) [inline README.md:59]: typos

**Review by KevinEady — CHANGES_REQUESTED** [review]
Saw this after approving

**Review by KevinEady — APPROVED** [review]
LGTM!

**Pull request merged**